src/main/resources/assets/soot/lang/zh_CN.lang

```diff
@@ -0,0 +1,27 @@
+embers.jei.recipe.dawnstone_anvil=黎明石砧合成
+embers.jei.recipe.alchemical_mixer=炼金台合成
+
+tile.ember_burst.name=Ember Spread Burst
+tile.ember_funnel.name=灰烬漏斗
+tile.alchemy_globe.name=炼金重组器
+
+item.signet_antimony.name=锑纹章
+
+fluid.sugar=熔融糖
+fluid.antimony=熔融锑
+fluid.boiling_wort=熔融麦芽汁
+fluid.boiling_potato_juice=熔融马铃薯汁
+fluid.boiling_wormwood=熔融Verdigris
```

Comment:
"The name verdigris comes from the Middle English vertegrez, from the Old French verte grez, an alteration of vert-de-Grèce ("green of Greece"). The modern French spelling of this word is vert-de-gris ("green of grey"). It was used as a pigment in paintings and other art objects (as green color), mostly imported from Greece (Grèce)."

There's also a "Verdigris Conconction" in Dragon's Dogma if you'd like to cross-reference translation but you could also just do a transliteration.